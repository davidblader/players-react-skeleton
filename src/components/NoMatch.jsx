import React from 'react';
import AnimalCrossingContainer from './AnimalCrossingContainer';
import AnimalCrossingHeader from './AnimalCrossingHeader';

const NoMatch = () =>
  (
    <AnimalCrossingContainer>
      <AnimalCrossingHeader>404</AnimalCrossingHeader>
    </AnimalCrossingContainer>
  );

export default NoMatch;
