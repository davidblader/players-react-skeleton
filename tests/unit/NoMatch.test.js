import React from 'react';
import { shallow } from 'enzyme';
import NoMatch from '../../src/components/NoMatch';

test('NoMatch shallow renders', () => {
  const component = shallow(<NoMatch />);
  expect(component).toMatchSnapshot();
});
