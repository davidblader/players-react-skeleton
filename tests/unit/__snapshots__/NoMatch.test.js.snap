// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMatch shallow renders 1`] = `
<AnimalCrossingContainer>
  <AnimalCrossingHeader>
    404
  </AnimalCrossingHeader>
</AnimalCrossingContainer>
`;
